### DISCUSSION TITLE
xpi code corruption?

### DISCUSSION THREAD

**Discussion by ndalbey** [retorquere/zotero-better-bibtex#2544] [Q&A] [answered]
Hi all,

Tried to download the better bibtex plugin but keep running into this error message: "The add-on downloaded from this site could not be added because it appears to be corrupt." Am I missing something, or is the file unavailable at this time?

Thanks!

**Comment by retorquere** [comment, marked as answer]
https://retorque.re/zotero-better-bibtex/support/faq/

**Reply by ndalbey** [reply]
Thank you! (...sorry I missed the FAQ section of the website).
